### PR TITLE
[Merged by Bors] - chore: remove porting note about adding `set_option linter.uppercaseLean3 false`

### PR DESCRIPTION
**Pull request by joneugster**
---

The lines `set_option linter.uppercaseLean3 false` together with the `#aligns` are already a left-over from the port. Marking some of them with an additional `-- porting note` is not necessary.

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Review by rwst — APPROVED** [review]
All is fine --- only porting note removals

File: Mathlib/Computability/DFA.lean [hunk 1]
```diff
@@ -24,7 +24,7 @@ open Computability
 
 universe u v
 
--- Porting note: Required as `DFA` is used in mathlib3
+-- Porting : Required as `DFA` is used in mathlib3
```
Comment (joneugster) [inline Mathlib/Computability/DFA.lean:27]:
```suggestion
```

**Review by Ruben-VandeVelde — APPROVED** [review]
maintainer merge

**Comment by riccardobrasca** [comment]
Thanks!

bors merge

**Pull request closed without merging**